src/flows/pipes/Notification/endpoints/AWS/index.ts

```diff
@@ -0,0 +1,112 @@
+import {TEST_NOTIFICATION} from 'src/flows/pipes/Notification/endpoints'
+import View from './view'
+import ReadOnly from './readOnly'
+
+export default register => {
+  register({
+    type: 'aws',
+    name: 'AWS SES Email',
+    data: {
+      url:
+        'https://email.your-aws-region.amazonaws.com/sendemail/v2/email/outbound-emails',
```

Comment:
I'm assuming this is just placeholder text right?

Reply:
yeah exactly